src/themes/greyVest/SearchLayout.js

```diff
@@ -1,13 +1,24 @@
-import React from 'react'
+import React, { useContext } from 'react'
 import PropTypes from 'prop-types'
-import { observer } from 'mobx-react'
+import { Observer, useLocalStore } from 'mobx-react'
 
-let SearchLayout = ({ mode, ...props }) => (
-  <div className={`gv-search-layout-${mode}`} {...props} />
-)
+let SearchLayoutContext = React.createContext()
+
+export let useSearchLayout = () => useContext(SearchLayoutContext)
+
+let SearchLayout = ({ mode = 'basic', ...props }) => {
+  let layout = useLocalStore(() => ({ mode }))
```

Comment:
This is a breaking change and makes `mode` into a modeDefault

Reply:
See my comments above on breaking changes